io.github.Sebicodes99.Prt.yml

```diff
@@ -0,0 +1,20 @@
+app-id: io.github.Sebicodes99.Prt
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: runner.sh
+finish-args:
+  - --share=network
+modules:
+  - name: PleaseRunThat
+    buildsystem: simple
+    build-commands:
+      - install -D Flatpak/runner.sh /app/bin/runner.sh
+      - install -D app.py /app/bin/app.py
+      - install -D Flatpak/.desktop /app/share/applications/io.github.Sebicodes99.Prt.desktop
+      - install -D Flatpak/plruth.png /app/share/icons/hicolor/128x128/apps/io.github.Sebicodes99.Prt.png
+      - install -D Flatpak/io.github.Sebicodes99.Prt.appdata.xml /app/share/metainfo/io.github.Sebicodes99.Prt.appdata.xml
+    sources:
+      - type: git
+        url: https://github.com/Sebicodes99/pleaserunthat-beta.git
+        tag: 1.6.0
```

Comment:
this tag doesn't exist either.

Reply:
Because I was just launching it, now it exists.